cypress-tests/cypress/e2e/rstudioSession.cy.ts

```diff
@@ -1,114 +1,77 @@
-        cy.contains("Back to Login").click()
-        cy.renkuLogin([{ username: email, password }])
-      }
-    })
-    // Login with the main dev deployment after registering on the CI deployment (if required)
-    cy.url().then((url) => {
-      if (url.includes("auth/realms/Renku/protocol/openid-connect/auth")) {
-        cy.renkuLogin([{ username: email, password }])
-      }
-    })
-    const httpBaseUrl = Cypress.config("baseUrl").replace(/^https:\/\//, "http://")
-    const httpsBaseUrl = Cypress.config("baseUrl").replace(/^http:\/\//, "https://")
-    cy.url().should(
-      "be.oneOf",
-      [
-        httpBaseUrl,
-        httpsBaseUrl,
-        httpBaseUrl + "/",
-        httpsBaseUrl + "/",
-      ]
-    )
-    cy.request("ui-server/api/user").its("status").should("eq", 200)
-  })
+    });
+  });
+
   afterEach(() => {
-    // Prevents the ui from automatically refreshing the page, it does this when the local storage is wiped
-    // and the local storage is wiped by default after an "it" section ends in cypress
-    cy.saveLocalStorage()
-  })
+    cy.logout();
+  });
   beforeEach(() => {
-    // Prevents the ui from automatically refreshing the page, it does this when the local storage is wiped
-    // and the local storage is wiped by default after an "it" section ends in cypress
-    cy.restoreLocalStorage()
-  })
-  it("Creates a project", { defaultCommandTimeout: 30000 }, () => {
-    const templateName = "Basic R"
-    cy.visit("/")
-    cy.get(`[data-cy=username-home]`).should("include.text", username)
-    cy.createProject(projectName, templateName, username)
-  })
-  it("Waits for the image to build", { defaultCommandTimeout: 300000 }, () => {
-    cy.waitForImageToBuild(username, projectName)
-  })
-  it("Launches a session", () => {
-    cy.startSession(username, projectName)
-  })
-  it("Waits for the session to fully start", { defaultCommandTimeout: 300000 }, () => {
-    cy.get(`.fullscreen-back-button`).should("be.visible").click()
-    cy.get(".time-caption").should("contain.text", "Running")
-  })
-  it("Opens the session in an iframe", () => {
-    cy.contains("Open").click()
-    cy.get("div.details-progress-box", { timeout: TIMEOUTS.vlong }).should("not.exist")
+    cy.robustLogin({ email, password, firstName, lastName });
+  });
+
+  it("Creates a project and launches a session", { defaultCommandTimeout: 30000 }, () => {
+    // Creates the project
+    const templateName = "Basic R";
+    cy.visit("/");
+    cy.get(`[data-cy=username-home]`).should("include.text", username);
+    cy.createProject(projectName, templateName, username);
+
+    // Waits for the image to build
+    cy.waitForImageToBuild(username, projectName);
+
+    // Launches a session
+    cy.startSession(username, projectName);
+
+    // Waits for the session to fully start
+    cy.get(`.fullscreen-back-button`).should("be.visible").click();
+    cy.get(".time-caption", { timeout: 60000 }).should("contain.text", "Running");
```

Comment:
There are a few places where I may have missed things. But I wanted to use one of the timeouts from `cypress-tests/config.ts` in cases like this. So just replace with `{ timeout: TIMEOUTS.vlong }`, or add another level because currently we have 10s, 30s and 300s for standard, long and vlong.